src/__tests__/typescript/hoc.tsx

```diff
@@ -22,4 +22,8 @@ function MyComponent({ id, size }: MyComponentProps) {
 
 const SizedMyComponent = withSize()(MyComponent)
 
-const foo = <SizedMyComponent id={1} />
+const onSize: WithSizeOnSizeCallback = ({ height, width }) => {
+  Boolean(height && width)
```

Comment:
Could you replace the body of this with the following:

```typescript
  if (width) {
    const foo = width + 1	    const foo = width + 1
  }	  }
  if (height) {	  if (height) {
    const foo = height + 1	    const foo = height + 1
  }	  }
  // typings:expect-error	  // typings:expect-error
  const h1 = height + 1	  const h1 = height + 1
  // typings:expect-error	  // typings:expect-error
  const w1 = width + 1	  const w1 = width + 1
```

This helps guard that width/height are nullable numbers.